source/configuration/read.txt

```diff
@@ -255,14 +240,14 @@ This partitioner is not compatible with hashed shard keys.
      - Description
```

Comment:
Add full name / path for all the partitioners:

`com.mongodb.spark.sql.connector.read.partitioner.PaginateBySizePartitioner`